src/App.js

```diff
@@ -7,18 +7,22 @@ import {useState} from 'react';
 // /*This is the app that will display the arrays of Object and the form with button*
 
 function App() {
-  const [newTodo, setNewTodo] = useState('');
+  const [todoList, setTodoList] = useState([]);
+  // const [todoList, setTodoList] = [];
```

Comment:
Unless you're keeping this around as a personal reference/note to remember how the project started, it's best to just delete unused code rather than commenting it out. To piggyback off of that, where is this `Search` component coming from? You've got it and its import commented out but they don't exist in your project.